css/style.css

```diff
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+    /* свойство scroll-behavior не наследуется, применяется к прокручиваемым блокам */

```

Comment:
it's supported very bad (https://caniuse.com/?search=scroll-behavior), read what to use instead

Reply:
Thanks!
I removed this css-property and used JS